src/vs/workbench/contrib/terminal/electron-browser/terminalInstance.ts

```diff
@@ -979,16 +1003,18 @@ export class TerminalInstance implements ITerminalInstance {
 		this._isExiting = true;
 		let exitCodeMessage: string;
 
-		if (exitCode) {
```

Comment:
`if (exitCode)` is fine here, we want the message when `exitCode !== undefined && exitCide !== 0`

Reply:
@Tyriar - The reason I did this is because it would be odd to see the word "process" when a custom execution never starts a process.

Reply:
But the only case it would be undefined is when `this._processManager.processState === ProcessState.KILLED_BY_USER` which is guarded later on